### PR TITLE
Update `prelogin.css` to a `.scss` file

### PR DESCRIPTION
**Pull request by nmajor25**
Shortcut Story ID: [sc-33392]

This was originally a `.css` file because the form iframe needed authentication. But now that it doesn't need authentication, the `prelogin.css` file can be made `.scss` without breaking the form styles.

The new `prelogin.scss` file is also updated to include sass code patterns we use elsewhere in the app.

### CODE REVIEW

File: src/js/views/globals/prelogin/prelogin.scss [hunk 1]
```diff
@@ -1,23 +1,3 @@
...
-
-@font-face {
-  font-family: 'ProximaSoft';
-  font-style: normal;
-  font-weight: bold;
-  src: url('/fonts/ProximaSoft/3955CC_1_0.woff2') format('woff2'),
-    url('/fonts/ProximaSoft/3955CC_1_0.woff') format('woff'),
-    url('/fonts/ProximaSoft/3955CC_1_0.ttf') format('truetype');
-}
-
```
Comment (nmajor25) [inline src/js/views/globals/prelogin/prelogin.scss]: I think this is good to remove...? The fonts look the same without it.

File: src/js/views/globals/prelogin/prelogin.scss [hunk 2]
```diff
@@ -85,13 +65,13 @@ body {
 }
 
 .prelogin__continue-button {
```
Comment (paulfalgout) [inline src/js/views/globals/prelogin/prelogin.scss:50]: This looks like it is reproducing our button classes.  Can we reuse the buttons module?

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/globals/prelogin/prelogin.scss (hunk 2 above)
Comment (nmajor25) [reply src/js/views/globals/prelogin/prelogin.scss:50]: Yup, good call. Just pushed an update that uses the buttons module.

Comment (paulfalgout) [reply src/js/views/globals/prelogin/prelogin.scss:50]: well I'd assume we could replace this class in the html with some `button-class`

File: src/js/views/globals/prelogin/prelogin.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
+body {
+  margin: 0;
+}
+
+.prelogin * {
```
Comment (paulfalgout) [inline src/js/views/globals/prelogin/prelogin.scss]: I bet this is covered by the core scss as well.

**Review by paulfalgout — CHANGES_REQUESTED** [review]

File: src/js/views/globals/prelogin/prelogin.scss (hunk 3 above)
Comment (nmajor25) [reply src/js/views/globals/prelogin/prelogin.scss]: After importing `'scss/provider-core.scss'`, I was able to remove this `.prelogin *` along with several other properties in the file that were duplicates from the core scss.

File: src/js/views/globals/prelogin/prelogin.scss (hunk 2 above)
Comment (nmajor25) [reply src/js/views/globals/prelogin/prelogin.scss:50]: The closest button class is:

```css
.button--large {
  @extend .button;

  font-size: 16px;
  padding: 8px 40px;
}
```

But it doesn't have the correct `margin-top` or `padding`.

Comment (paulfalgout) [reply src/js/views/globals/prelogin/prelogin.scss:50]: ah I didn't realize `.button--green` was already in the html.. clearly wasn't doing anything before.

**Review by paulfalgout — APPROVED** [review]

File: src/js/views/globals/prelogin/prelogin.scss (hunk 2 above)
Comment (nmajor25) [reply src/js/views/globals/prelogin/prelogin.scss:50]: Correct, before `'scss/modules/buttons.scss';` was imported, it wasn't doing anything.

**Pull request merged**